docs/profiling.md

```diff
@@ -0,0 +1,27 @@
+# Profiling
+
+![Example](img/profiler.png)
+
+It's possible to get a full profile of Boa in action.  
+Sometimes this is needed to figure out where it is spending most of it's time.
+
+We use a crate called [measureme](https://github.com/rust-lang/measureme), which helps us keep track of timing functions during runtime.
+
+## How To Use
+
```

Comment:
We should add fill in the how to use section

Reply:
The how-to is under the prerequesites, ive reodered now